### ISSUE TITLE
Bump support libs to 23.3

### ISSUE THREAD

**Issue by AndyScherzinger** [owncloud/android#1557]
I would like to bump the support libs to v23.3
This is basically a gradle change plus replacing the corresponding ant dependencies in the lib folder plus some extra files in there to make the ant build work. 
Exemplary thing we would get:
- vector drawables - which might only be usable when we drop ANT support, so no benefit for now
- the recycler view implementation has been updated to also support wrap_content and auto layouting (though we don't use the recycler view yet)
- we can than add a day/night theme if we want to
- custom tab as in better support to open a webpage within the app itself

Details: http://android-developers.blogspot.de/2016/02/android-support-library-232.html

Any opinion on this? @davivel @tobiasKaminsky @przybylski @malkomich

**Comment by tobiasKaminsky** [comment]
I would do the upgrade when we upgrade to recycle view.
Just upgrading the lib without a real benefit is not worth enough the testing, I think

**Comment by AndyScherzinger** [comment]
That is true :) My idea was to do such things right after a release so this would tested implicitly (I know that doesn't mean "tested"... just to not have this discussion ;) - I just proposed this since I opened a PR a while ago with the v23 update which also included all the fixes necessary due to deprecations. This PR has already been integrated since it also contained AppCompat bugfixes  which we needed (not the original purpose why I opened that PR). 
So I see the issue with testing too but I also see the benefit of staying up to date with the support lib releases.
